### PR TITLE
Add Docker Image for Cisco Meraki v1.46.0

### PR DESCRIPTION
**Pull request by NoelKocheril**
## Status
Ready

## Related Content Pull Request
Related PR: [link to the PR at demisto/content](https://github.com/demisto/content/pull/35048)

## Related Issues
Related: link to the issue

## Description
Adding Docker Image for Cisco Meraki Python Package

### CODE REVIEW

File: Pipfile.lock
Comment (BEAdi) [inline Pipfile.lock:1]: Are those changes needed? Please revert them.

File: docker/cisco-meraki/Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+        "sources": [
+            {
+                "name": "pypi",
+                "url": "https://pypi.org/simple",
+                "verify_ssl": true
+            }
+        ]
+    },
+    "default": {
+        "aiohttp": {
```
Comment (BEAdi) [inline docker/cisco-meraki/Pipfile.lock:19]: The `aiohttp` package of all versions is vulnerable to DoS.
It has a CVE that need to be mitigated.

File: docker/cisco-meraki/Pipfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+[packages]
+"ruamel.yaml" = "*"
+requests = "*"
+dockerfile_parse = "*"
+pyyaml = "*"
+regex = "*"
+python-dateutil = "*"
+gitpython = "*"
+meraki="1.46.0"
```
Comment (BEAdi) [inline docker/cisco-meraki/Pipfile:14]: Change it to be `"*"` or add a comment explaining why we are closing this specific version.

**Review by BEAdi — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please address my important comments.
Please provide the related [content](https://github.com/demisto/content) PR that requires the use of this docker image. It's necessary for us in order to merge this PR.

**Comment by NoelKocheril** [comment]
> Thank you for your contribution! Please address my important comments. Please provide the related [content](https://github.com/demisto/content) PR that requires the use of this docker image. It's necessary for us in order to merge this PR.

I'm still in the process of creating the PR for the Integration, happy to create an initial PR as all of the work is on my local dev environment

File: docker/cisco-meraki/Pipfile (hunk 2 above)
Comment (NoelKocheril) [reply docker/cisco-meraki/Pipfile]: No particular reason, version 1.46.0 was the version I was testing with

File: Pipfile.lock
Comment (NoelKocheril) [reply Pipfile.lock:1]: Accidental change when updating the PIPENV setup, reverted to last commit by demistobot

**Comment by NoelKocheril** [comment]
> > Thank you for your contribution! Please address my important comments. Please provide the related [content](https://github.com/demisto/content) PR that requires the use of this docker image. It's necessary for us in order to merge this PR.
> 
> I'm still in the process of creating the PR for the Integration, happy to create an initial PR as all of the work is on my local dev environment

Created an initial PR in Demisto/content for the updated integration

https://github.com/demisto/content/pull/35048

**Comment by BEAdi** [comment]
Thank you for your effort in developing the content PR! However, we regret to inform you that we are unable to accept it at this time. We are currently in the process of integrating the same API, which is scheduled for release in the coming months.

Your contribution is valued, and we encourage you to continue participating. We look forward to your future commits. Please feel free to reach out if you have any questions. Thank you once again!

**Pull request closed without merging**